express/properties/non_scalar/symmetry.py

```diff
@@ -6,15 +6,26 @@ class Symmetry(NonScalarProperty):
     Symmetry property class.
     """
 
-    def __init__(self, name, parser, *args, **kwargs):
-        super(Symmetry, self).__init__(name, parser, *args, **kwargs)
+    def __init__(self, name, parser, is_non_periodic=False, *args, **kwargs):
+        super(Symmetry, self).__init__(name, parser, is_non_periodic, *args, **kwargs)
+        self.is_non_periodic = is_non_periodic
 
     def _serialize(self):
-        return {
-            "spaceGroupSymbol": self.parser.space_group_symbol()["value"],
-            "tolerance": {
-                "value": self.parser.space_group_symbol()["tolerance"],
-                "units": "angstrom"
-            },
-            "name": self.name
-        }
+        if self.is_non_periodic == False:
+            return {
+                "spaceGroupSymbol": self.parser.space_group_symbol()["value"],
+                "tolerance": {
+                    "value": self.parser.space_group_symbol()["tolerance"],
+                    "units": "angstrom"
+                },
+                "name": self.name
+            }
+        else:
+            return {
+                "pointGroupSymbol": self.parser.point_group_symbol()['value'],
+                "tolerance": {
+                    "value": self.parser.point_group_symbol()["tolerance"],
+                    "units": "angstrom"
+                },
+                "name": self.name
+            }
```

Comment:
Per https://github.com/Exabyte-io/esse/pull/191/files#r779988337 - both symmetries can be present simultaneously